code/model/SubgoalPolicy.cpp

```diff
@@ -1990,9 +1990,7 @@ void SubgoalPolicy::SampleSubgoalSequence (const Problem& _rProblem,
       		size_t i_QueryIndex = s_QuestionString.find(dq_QuestionArgs[2]);
       		String s_QuestionQuery = s_QuestionString.substr(i_QueryIndex);
 
-          if(false == AskQuestion(s_QuestionType, s_QuestionQuery)) {
-            //TODO cout error, throw error type behavior
-          }
+          assert(true == AskQuestion(s_QuestionType, s_QuestionQuery))
```

Comment:
Eventually, we want to run this outside of debug mode, so we should have this outside of an assert